src/gen_module.sh

```diff
@@ -33,6 +34,20 @@ if [ ! -e ${blender_bin} ]; then
     exit 1
 fi
 
+# check if PYTHON_BIN binary is availble
+if ! command -v ${PYTHON_BIN} > /dev/null; then
+    echo "Error: Cannot find ${PYTHON_BIN} binary."
+    exit 1
+fi
+python=$(command -v ${PYTHON_BIN})
+
+# check if python version meets our requirements
+IFS=" " read -r -a pythonversion <<< "$(${python} -c 'import sys; print(sys.version_info[:])' | tr -d '(),')"
```

Comment:
I think `python_version` would be better  variable name.

Reply:
renamed